yarn-project/end-to-end/src/e2e_fees/private_refunds.test.ts

```diff
@@ -136,23 +149,32 @@ describe('e2e_fees/private_refunds', () => {
     const aliceRandomness = Fr.random(); // Called user_randomness in contracts
```

Comment:
This test is not really finished because I can't check balances etc. because of [#7717](https://github.com/AztecProtocol/aztec-packages/issues/7717).